config.js

```diff
@@ -134,6 +134,15 @@ config.iota = {
      */
```

Comment:
As far as I understand, this PR is about modification in the docker part. The config.js file is not used by the new entry point file. Thus, let's keep it simple are avoid chaning the config.js file.

Reply:
@fgalan - I thought the `ENV` variables only overwrote *existing* attributes in the `config.js` file. If adding this is a necessary part of this PR since without it the Username/Password, secure flag & etc. won't get set.

Could you clarify please?

Reply:
Not 100% sure but looking to coce like this https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/commonConfig.js#L144 it seems the env var processing logic is able to create what can be missing from config.js.

Another configuration would be great. @dcalvoalonso what do you think?

Reply:
Awaiting for response

Reply:
@jason-fox did a test and it works without that addition. However, as we has been dicussing in the "twin" PR maybe is a good idea to include the block but in a commented way. See details at https://github.com/telefonicaid/iotagent-ul/pull/339#discussion_r260704766

Thus, in conclusion, please add here the same block it as added it in IOTA-UL (copy-paste basically ;)

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/9a74a9a0480f11abfd56eae8461da68ebbb54585